recipes-mender/mender/mender_0.1.bb

```diff
@@ -36,8 +36,18 @@ do_compile() {
   PATH="${B}/bin:$PATH"
   export PATH
 
-  cd ${S}
-  godep go build -o ${B}/mender
+  # mender is using vendored dependencies, any 3rd party libraries go to into
+  # /vendor directory inside mender source tree. In order for `go build` to pick
+  # up vendored deps from our source tree, the mender source tree itself must be
+  # located inside $GOPATH/src, for instance $GOPATH/src/mender
+  #
+  # recreate temporary $GOPATH/src/mender structure and link our source tree
+  mkdir -p ${B}/src
+  test -e ${B}/src/mender || ln -s ${S} ${B}/src/mender
+  cd ${B}/src/mender
+
+  # run verbose build, we should see which dependencies are pulled in
+  oe_runmake V=1
```

Comment:
I think this doesn't preserve the `-o` functionality. This is important, because if using the Yocto `externalsrc` functionality, the ${S} will be in my home directory, and I don't want it to write there.